Modules/Filtering/LabelMap/include/itkStatisticsLabelMapFilter.h

```diff
@@ -137,9 +137,11 @@ class ITK_TEMPLATE_EXPORT StatisticsLabelMapFilter
   static constexpr unsigned int
   GetDefaultNumberOfBins()
   {
-    return NumericTraits<FeatureImagePixelType>::IsInteger && sizeof(FeatureImagePixelType) <= 2
-             ? 1 << (8 * sizeof(FeatureImagePixelType))
-             : 128;
+    constexpr unsigned short bitsShift =
+      std::min(8 * sizeof(typename Self::FeatureImagePixelType), sizeof(unsigned int) - 1);
+
+    return NumericTraits<FeatureImagePixelType>::IsInteger && sizeof(FeatureImagePixelType) <= 2 ? 1u << bitsShift
+                                                                                                 : 128;
```

Comment:
It may be for style reasons only, but how about using `128u` for symmetry with `1u`?

Reply:
done